js/index.js

```diff
@@ -0,0 +1,36 @@
+/* global $ */
+
+//PSEUDOCODE
+
+//prevent form's default function
+//when user clicks "update" 
+//take user input and store it in a variable
+//if var = NYC, New York City, or New York
+//then display background image of NYC
+//else if var = San Francisco, SF, or Bay area
+//then display SF background image
+// if...then...else... LA...Austin...Sydney
+
+
+function cityInput(event) {
+  event.preventDefault();
+  var city = $('#city-type').val();
+  if (city === 'NYC' || city === 'New York' || city === 'New York City') {
+    $('body').removeClass();
+    $('body').addClass('nyc');
```

Comment:
Not a big deal but one thing you could have done is chain these calls since they both are happening to `body`

```js
$('body').removeClass().addClass('nyc');
```